lib/pact_broker/badges/service.rb

```diff
@@ -40,16 +40,20 @@ def can_i_deploy_badge_url(tag, environment_tag, label, deployable)
         build_shield_io_uri(title, status, color)
       end
 
-      def can_i_merge_badge_url(version_number: nil, deployable: nil)
+      def can_i_merge_badge_url(deployable: nil)
         title = "can-i-merge"
-        status = deployable ? "yes" : "no"
-        if deployable.nil?
-          color = "lightgrey"
-          status = "unknown"
+        
+        # rubocop:disable Layout/EndAlignment
```

Comment:
rubocop was complaining that, case is not aligned with end, but here I am trying to return the value from case into LHS variables